api/lib/resolvers.js

```diff
@@ -280,7 +280,7 @@ const resolvers = (networkList, notificationController) => ({
         .map((network) => {
           // the server side nodes are mostly whitelisted or secret so we don't want to send them to the FE
           return Object.assign({}, network, {
-            rpc_url: network.public_rpc_url,
+            rpc_url: network.public_rpc_url || network.rpc_url,
```

Comment:
```suggestion
            rpc_url: network.public_rpc_url
```

the comments states that we don't want to pass the private rpc out

Reply:
there are some networks in staging-DB that lack of `rpc_url`, like cosmos for example. Guess we could use nylira there, but what about the others :shrug:

Reply:
is this a problem? you only had the issue with Polkadot, right?

Reply:
yeah, true. Which left me wondering. OK, let me try this PR without that bit and see if it still works

Reply:
Yeah, it works flawlessly. So the real fix was just adding `public-rpc-url` to the query body actually